Makefile

```diff
@@ -133,14 +135,19 @@ else
  detected_OS := $(strip $(shell uname))
 endif
 
-installganache: 
-ifeq ($(RLN), true)
+installganache:
+ifeq (true, true); # do nothing
+else ifeq ($(RLN), true) 
+	npm install ganache-cli; npx ganache-cli -p	8540	-g	0	-l	3000000000000&
+else  ifeq ($(CI), true)
 	npm install ganache-cli; npx ganache-cli -p	8540	-g	0	-l	3000000000000&
 endif
 
 rlnlib:
 ifeq ($(RLN), true)
 	cargo build --manifest-path vendor/rln/Cargo.toml
+else  ifeq ($(CI), true)
```

Comment:
It seems a bit odd to me that we now have:

- RLN flag, off by default
- RLN is built in CI env
- But tests are commented out due to Windows Ganache issue

Wouldn't something like this make more sense:
- RLN flag is off by default
- Ganache tests are dealt with separate flag
- Ganache flag/behaviour is off for Windows CI (or all CI if it it isn't straightforward to detect OS (?))

Reply:
Sure, I introduced a new flag `ONCHAIN_RLN` that controls rln-relay tests that need interaction with the `ganache-cli` process. Now we have:
- `RLN` and  `ONCHAIN_RLN` set off by default
- CIs: `RLN` is on and  `ONCHAIN_RLN` is off